### DISCUSSION TITLE
The universe was a bad idea

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#245] [Metaphysics and Philosophy]
(Posting this in metaphysics because it's more relevant here than in the z=14 discussion...)

Reality, existence, the world, the universe, the cosmos... is there actually a meaningful distinction between these things?

It's easier to say "the universe is 13.8 billion years old" than "existence as we know it about 3 times older than the rock we live on".

The latter is what's being claimed, but no one says it out loud because of how obviously dumb it is. Call it "the universe" though, and we ask, when's its birthday? What's its sign? I hope it's a Leo because I am too!

We basically give it a different name and it automatically becomes easier to talk about like some popular culture celebrity. But it's nonsense none the less.

I woke up with a bit of a short fuse today, so that's what's behind this. But I can't help but think of the fella (Andrew Nimmo) that came up with "multiverse". He was at odds with the idea of people talking about "parallel universes." So he said, there's can't be parallel universes. There can multiverses, many of them, but only one "universe." Cause words mean things.

People liked that idea so they adopted it, but in the exact opposite way that was intended. There is one multiverse and many universes. Serves him right for to trying to use words in a meaningful way. What actually happens is no different than the prisoner's of Plato's cave... people just say what they think is good and whenever someone points out they're just making stuff up arbitrarily, that person is bad.

Anyway, the universe is a silly idea.

"In the beginning the Universe was created. This has made a lot of people very angry and been widely regarded as a bad move."

**Comment by HanDeBruijn** [comment]
Relevant reply in [thread 131](https://github.com/orgs/a-cosmology-group/discussions/131#discussioncomment-6823263), comment in [thread 243](https://github.com/orgs/a-cosmology-group/discussions/243#discussioncomment-9908284).
It effectively means that **there is no universe**, in the sense that there would exist something that contains everything.

**Reply by mikehelland** [reply]
To quote @RedshiftDrift :

"Redefining a word only causes greater confusion."

The universe is simply a redefinition of Nature. It hasn't gone well.

**Reply by mikehelland** [reply]
@budrap00 makes points that get "confused" emoji's. I get it though.

**Reply by RedshiftDrift** [reply]
"Above all we must not overclaim for this fascinating subject [cosmology] which, it can be argued, is not a proper science at all. Rutherford for instance said *“Don’t let me hear anyone use the word ‘Universe’ in my department”*." [^1]

Disney goes on:
> Shouldn’t we scientists be saying something like this to the general public:
>> “It is not likely that we primates gazing through bits of glass for a century or two will dissemble the architecture and history of infinity. But if we don’t try we won’t get anywhere. Therefore we professionals do the best we can to fit the odd clues we have into some kind of plausible story. That is how science works, and that is the spirit in which our cosmological speculations should be treated. Don’t be impressed by our complex machines or our arcane mathematics. They have been used to build plausible cosmic stories before - which we had to discard afterwards in the face of improving evidence. The likelihood must be that such revisions will have to occur again and again and again.”

[^1]: M. J. Disney, “The Case Against Cosmology,” General Relativity and Gravitation, vol. 32, no. 6, pp. 1125–1134, Jun. 2000, [arXiv:astro-ph/0009020](https://arxiv.org/abs/astro-ph/0009020) doi: [10.1023/A:1001981929727](https://doi.org/10.1023/A:1001981929727).

**Reply by budrap00** [reply]
I've had a suspicion for some time that as a reward for his efforts Disney got "Arped". The brief Wiki entry for him contains this one sentence paragraph:
>Disney was a professor at Cardiff University until his retirement in 2005. Disney is an outspoken critic of the theory of cosmic inflation.

There is no current information given for him though he appears to be alive and the Cosmology paper is not referenced. Describing him as an outspoken critic of cosmic inflation is like describing an atheist as a critic of Judaism. 

At the time of his retirement he would have been 68; full professors usually carry on longer than that with the benefit of a reduced workload. So the question is did he retire or was he retired?

**Reply by mikehelland** [reply]
I've encountered his writing before, specifically this:

https://www.americanscientist.org/article/modern-cosmology-science-or-folktale

Which is from 2007, so he continued to write for at least a little longer after retirement.

**Reply by mikehelland** [reply]
https://mjdisney.org/

>So when I was forcibly retired at age 68 I lost my research assistants, my students, my ability to travel freely etc… thus most of the interactions I had so relied upon to do cutting-edge science.

**Reply by LifeIsStrange** [reply]
His website intro is amazing

**Reply by budrap00** [reply]
He got Arped all right and it looks like an ongoing project; the [Wiki stub](https://en.wikipedia.org/wiki/Mike_Disney) has a permanently dead link for his web site. Does anybody here do Wiki editing? Updating that link and adding some material about the Cosmology paper and his forced retirement would be a good way to stick it to, or at least call out, the high priests guarding the sacred temple of modern cosmology.

**Reply by mikehelland** [reply]
I've written a wikipedia article. The thing to do would be find any books and news articles that mention him. If we could gather some of that, I'd be happy to put something together.

**Reply by budrap00** [reply]
What I had in mind was just a few factual corrections that wouldn't require any research, such as fixing the broken web site link, changing "_Disney is an outspoken critic of the theory of cosmic inflation_" to "_Disney is a critic of the standard model of cosmology_" and providing a supporting link to the 2000 Cosmology paper.

A more robust biographical sketch would require a fair amount of research but those quick and defensible fixes would be a good starting point and at least set the record straight.

**Reply by mikehelland** [reply]
It all come down to Wikipedia's standard of being "encyclopedic." If we could find some books or articles not written by Disney that showed his involvement with the Hubble Space Telescope, that would be much easier to get in, and the tweaks you describe would be easier to accept along with it. 

edit: I think I've got a good start on this:

https://www.google.com/books/edition/The_Oxford_Handbook_of_the_History_of_Mo/OsKKDwAAQBAJ?hl=en&gbpv=1&dq=Michael+Disney+cosmology&pg=PA157&printsec=frontcover

**Comment by OlliSant** [comment]
It is the question of the whole and the parts. Of course the whole exists too, the universe, it exists in a different way, and is a different entity as its parts. 

If we don't know the whole, we don't know the parts in the right way, and only by knowing the parts, we know the whole. (Hegel). 

There is things in the whole that we don't yet know, and in the parts too. But we can make theories of the whole, the theory of the universe, cosmology, and of parts, of galaxies and forces and so on, physics, biology, psychology, sociology, theology. And test the theories and choose the best theories, and change them better and better. The reality remains the same whole the time, that is realism in philosophy. We change the theories better and better, but not the reality as in BB and in any stagnation in science. 

This is the science and philosophy, and there is no unity in science and philosophy, only different theories. But the reality remains the same. We should yet accept some different theories in cosmology, not only one.

**Reply by mikehelland** [reply]
Sure, we could just call it physical reality, or space, time and matter. The universe seems to be a newer term. Newton, for example, spoke about "nature" mostly.

The universe is easier to anthropomorphize, and can lead some to think it's a common physical object like anything else. Its use hasn't really clarified anything in particular, but gives us lots of opportunity for misuse and confusion. 

It's not a big deal, ultimately, what we call the "whole." Just something that's been somewhat of an irritation for me as the years go on. I'm glad I brought it up though. I particularly like the Rutherford quote: "Don’t let me hear anyone use the word ‘Universe’ in my department”.

**Reply by LifeIsStrange** [reply]
The whole isn't necessarily made only if it's parts, and therefore if by universe or spacetime we want to denote what exists besides matter and radiation, it is meaningful.
This is basically a discussion about the supposed intrinsic properties of spacetime, whether we talk about it's geometry or its "forces" the technical term for those properties is the aether and general relativity is in the strictest sense of the term, an aether theory.

**Reply by budrap00** [reply]
It is not the word "universe" that is the problem. It is the underlying concept of a unitary, simultaneously-existing entity that encompasses the "whole" or "everything" or whatever, that is the problem. That concept has no scientific basis. 

If you want to discuss that concept of "everything" philosophically, mathematically, theologically, etc., that's fine but it has nothing to do with science which is the study of the phenomenological world and that world, by its nature does not contain a unitary or unifying, simultaneously-existing entity. In physics there ain't no such animal.

**Reply by budrap00** [reply]
>...if by universe or spacetime we want to denote what exists besides matter and radiation, it is meaningful.

Neither universe or spacetime are observed phenomenon; they are products of the human imagination. Denoting something that's not there isn't supposed to be part of the scientific method. It is exactly that methodological error that has produced the two absurd and nonsensical standard models. You can't fix this problem by employing the same method that caused the problem to begin with.

All that exists scientifically speaking _are_ matter and electromagnetic radiation. All the theoretical labors of the past half century have been in service of not dealing with that basic fact.

**Reply by mikehelland** [reply]
> unitary or unifying,

I'm right there with you. Not only does "universe" give us a word for which we already had many, it throws that "uni" prefix in there, giving it monotheistic overtones as well.

**Comment by OlliSant** [comment]
No, there is such entity as the universe. It is eternal and boundless. And it has a definite size and no outside place exists.

We must have such consept in the theory of the universe, in cosmology. It just exists in a different way as any other thing. We have it in  our theories, and theory must be of it, not of observable universe.

If we don't have it experimentally, objectively, it is OK. This is rational thinking, theoretical thinking and it is OK. Empiricism is wrong. In cosmology and in philosophy.

Science is to make theories better and better, to test them.

**Comment by OlliSant** [comment]
The problem is solved by making two theories. One for the whole universe, and one for our local subuniverse. 

Both exist.

https://www.santavuori.com/Galaks2.pdf
